usr/share/ublue-os/firstboot/yafti.yml

```diff
@@ -67,24 +74,25 @@ screens:
           packages:
             - Bottles: com.usebottles.bottles
             - Discord: com.discordapp.Discord
-            - Games: org.gnome.Games
+            - GNOME Games: org.gnome.Games
             - Heroic Games Launcher: com.heroicgameslauncher.hgl
-            - MangoHUD: org.freedesktop.Platform.VulkanLayer.MangoHud//22.08
             - Steam: com.valvesoftware.Steam
-            - Gamescope for Steam: com.valvesoftware.Steam.Utility.gamescope
+            - Gamescope (Utility): com.valvesoftware.Steam.Utility.gamescope
+            - MangoHUD (Utility): org.freedesktop.Platform.VulkanLayer.MangoHud//22.08
+            - SteamTinkerLaunch (Utility): com.valvesoftware.Steam.Utility.steamtinkerlaunch
             - Proton Updater for Steam: net.davidotek.pupgui2
         Office:
-          description: Bow to Capitalism
+          description: Boost your productivity.
```

Comment:
Revert this change NOW :gun:. Oh shoot, that's a water pistol... Well, wouldn't you agree that "Bow to Capitalism" and "Boost your productivity" mean the same thing with a difference in personality...

Reply:
@EinoHR Yeah it was a bit funny, but we should be neutral and set a good example. :)